.github/workflows/build_and_test.yml

```diff
@@ -7,10 +7,6 @@ on:
       - release
   pull_request:
 
-defaults:
-  run:
-    shell: bash -euxo pipefail {0}
-
```

Comment:
Is it removed for any particular reason?

Reply:
Yes as kaniko & aws-cli don't have bash, I the run-shell for the jobs that need it